inlong-sort/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/multiple/IcebergSingleStreamWriter.java

```diff
@@ -100,20 +103,6 @@ public void open(Configuration parameters) throws Exception {
         this.taskWriterFactory.initialize(subTaskId, attemptId);
         // Initialize the task writer.
         this.writer = taskWriterFactory.create();
-
-        // Initialize metric
-        MetricOption metricOption = MetricOption.builder()
-                .withInlongLabels(inlongMetric)
-                .withInlongAudit(auditHostAndPorts)
-                .withInitRecords(metricState != null ? metricState.getMetricValue(NUM_RECORDS_OUT) : 0L)
-                .withInitBytes(metricState != null ? metricState.getMetricValue(NUM_BYTES_OUT) : 0L)
-                .withInitDirtyRecords(metricState != null ? metricState.getMetricValue(DIRTY_RECORDS_OUT) : 0L)
-                .withInitDirtyBytes(metricState != null ? metricState.getMetricValue(DIRTY_BYTES_OUT) : 0L)
-                .withRegisterMetric(RegisteredMetric.ALL)
-                .build();
-        if (metricOption != null) {
-            metricData = new SinkMetricData(metricOption, getRuntimeContext().getMetricGroup());
-        }
     }
```

Comment:
Maybe this will cause single table sync lost metric when remove it

Reply:
https://github.com/apache/inlong/blob/34a7e8f501e9edde01a48b4413fcac3770644536/inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/metric/sub/SinkTableMetricData.java#L101~L127

Reply:
@lordcheng10 single table sync will exec FlinkSink#chainIcebergSingleSinkOperators#createStreamWriter#IcebergSingleStreamWriter. So metric will and dirty data process logic will lost